server/src/app/views.py

```diff
@@ -24,55 +23,50 @@
 # Test route
 #########################################################################################
 @app.route('/',methods=['GET'])
-def verifyPortListening():
+def verify_port_listening():
     return 'Listening on Port 3001 for the Server!'
 
 #########################################################################################
 # Authorize routes
 #########################################################################################
 
 @app.route('/api/authorize/authurl',methods=['GET'])
-def getAuthUrl():
+def get_auth_url():
     """ DEVELOPER NOTE:
     * to utilize the latest security features/best practices
     * it is recommended to utilize pkce
     """
     # get configuration and settings
-    myEnv = request.args.get('env') or 'development'
-    myVersion = request.args.get('version') or 'v2'
+    my_env = request.args.get('env') or 'development'
+    my_version = request.args.get('version') or 'v2'
     PKCE = request.args.get('pkce') or True
-
-    settings = Settings(myEnv,myVersion,PKCE)
-
-    configSettings = getConfigSettings(myEnv)
-    authUrl = generateAuthorizeUrl(settings, configSettings)
-    return authUrl
+    return generate_authorize_url(Settings(my_env, my_version, PKCE), get_config_settings(my_env))
 
 @app.route('/api/authorize/currentAuthToken',methods=['GET'])
-def getCurrentAuthToken():
+def get_current_auth_token():
```

Comment:
Nice job changing the style to match the language. This also makes me think of @dtisza1 suggestion to add style enforcing (like black, etc.) and maybe we can get that in a future release. Appreciate you making these updates here!